.gitlab/junit_upload/functional_test_junit_upload_system_probe.yml

```
@@ -29,6 +29,8 @@ functional_test_junit_upload_system_probe:
   tags: ["arch:amd64"]
   allow_failure: true
   when: always
+  rules:
+    - !reference [.except_mergequeue]
```

```suggestion
    - !reference [.except_mergequeue]
    - when: on_success
```
I think we need this (at least to be consistent with the other place it's defined)

---

Won't this mean that the job doesn't run if previous stages failed ? i.e. we would only upload when everything is green ?

---

🤔 I hate `rules`, that seems correct reading it here - if that's the case we'll want to update that elsewhere.

---

I updated the other functional tests upload jobs, I think this makes it consistent

The regular upload jobs don't have this issue AFAIU